packages/frontend/src/components/common/Filters/FilterInputs/AutoComplete/AutoComplete.tsx

```diff
@@ -60,43 +67,20 @@ const AutoComplete: FC<Props2> = ({
         [value],
     );
 
-    const renderCreateOption = useCallback(
-        (
-            q: string,
-            active: boolean,
-            handleClick: React.MouseEventHandler<HTMLElement>,
-        ) =>
-            !isSearching ? (
-                <MenuItem2
-                    icon="add"
-                    text={`Add "${q}"`}
-                    active={active}
-                    onClick={handleClick}
-                    shouldDismissPopover={false}
-                />
-            ) : (
-                <StyledSpinner />
-            ),
-        [isSearching],
-    );
     return (
         <Suggest2
             className={disabled ? 'disabled-filter' : ''}
             disabled={disabled}
             fill
-            items={Array.from(options).sort((a, b) =>
-                a.localeCompare(b, undefined, { sensitivity: 'base' }),
-            )}
-            noResults={
-                isFetchingInitialData ? (
-                    <StyledSpinner />
-                ) : (
-                    <MenuItem2 disabled text="No suggestions." />
-                )
-            }
-            itemsEqual={(v, other) => v.toLowerCase() === other.toLowerCase()}
+            items={results}
+            itemsEqual={(a, b) => a.toLowerCase() === b.toLowerCase()}
```

Comment:
fyi: @owlas recently changed this to `a.localeCompare(b, undefined, { sensitivity: 'base' }),`